docs/generated/packages/vite/executors/test.json

```diff
@@ -57,6 +57,10 @@
       "reportsDirectory": {
         "type": "string",
         "description": "Directory to write coverage report to."
+      },
+      "testFile": {
+        "description": "The name of the file to test.",
+        "type": "string"
```

Comment:
the option is called `testNamePattern` (aliased to `-t`) in the vitest cli docs
we should use that name instead as the executor options should closely mimic the CLI of the tool they are wrapping.
https://vitest.dev/guide/cli.html#options

```suggestion
      "testFilePattern": {
        "aliases": ["t"],
        "description": "Run tests with full names matching the pattern",
        "type": "string"
```

Reply:
note: this file will auto update when the schema.json file is updated and `yarn documentation` is updated.

Reply:
`@nrwl/vite:test` already has an option called [testNamePattern](https://nx.dev/packages/vite/executors/test#testnamepattern), which does map to the vitest `testNamePattern`. `testNamePattern` doesn't fully achieve the goal of running a single (or multiple) test files, as it looks at the test names (e.g. inside the `describe`, or `it` block), rather than the file names.

I wasn't able to find `testFilePattern` in the vitest docs.

Reply:
One potential solution is to pass `options._` to `startVitest` like this `const ctx = await startVitest(options.mode, options._ ?? [], settings);`.

if I run `nx test my-lib myTest.spec.ts` the `options` object looks like this
```js
{
  passWithNoTests: true,
  reportsDirectory: '../../coverage/packages/my-lib',
  _: [ 'myTest.spec.ts' ],
  mode: 'test',
  watch: false,
  update: false,
  coverage: false
}
```

I'm just not sure how this could be documented.

Reply:
Ah I didn't see that. 
yeah okay that makes sense. Looks like Vitest just passes in the 'rest of the args' to the `startVitest` function. 

https://github.com/vitest-dev/vitest/blob/main/packages/vitest/src/node/cli.ts#L50-L52

having a --testFile arg I think probably makes the most sense to me in this case as passing _ is probably not the best move as it'll most likely be subject to changing over time and breaking people without notice. 

any objections @mandarini?

Reply:
Hey there @barbados-clemens @mandarini, just checking in to see if there are any updates on this PR?

Reply:
@jaysoo could you take look at this PR since @mandarini is out of the office?